### PR TITLE
Add Support for Extra Labels

### PR DESCRIPTION
**Pull request by sundereshwar**

### CODE REVIEW

File: Examples/Histogram/Parser.go [hunk 1]
```diff
@@ -30,6 +33,11 @@ func main() {
 	fmt.Println(histmap)
 	var ht Export
 	ht.Field1 = histmap
-	result := prometheus_handler.GenericPromDataParser(ht)
+	addToHandler = append(addToHandler, struct {
+		Structure interface{}
+		Datatype  string
+		Labelname string
```
Comment (manishasalve) [inline Examples/Histogram/Parser.go:39]: Datatype => DataType
Labelname => LabelName 

minor suggestions.. we should follow camel case style.

File: src/prometheus_handler.go [hunk 2]
```diff
@@ -67,7 +74,7 @@ func makePromCounter(label string, count string) string {
 	return entry + "\n"
 }
 
-func makePromHistogram(label string, histogram map[string]int) string {
+func makePromHistogram(label string, histogram map[string]int, datatype string, labelname string) string {
```
Comment (manishasalve) [inline src/prometheus_handler.go:77]: Same here:
datatype => dataType
labelname => labelName

File: src/prometheus_handler.go [hunk 3]
```diff
@@ -80,7 +87,7 @@ func makePromHistogram(label string, histogram map[string]int) string {
 	sort.Strings(bounds)
 
 	for _, bound := range bounds {
-		entry := fmt.Sprintf(`%s_bucket{le="%s"} %d`, label, bound, histogram[bound])
+		entry := fmt.Sprintf(`%s_bucket{le="%s", %s="%s"} %d`, label, bound, strings.TrimRight(labelname, ".bt"), datatype, histogram[bound])
```
Comment (manishasalve) [inline src/prometheus_handler.go:90]: Can you make sure we maintain max 80 character length and if it exceed , move the statement to next line.

File: src/prometheus_handler.go [hunk 4]
```diff
@@ -97,33 +104,38 @@ func makePromGauge(label string, value string) string {
...
+					count := parseCounter(fieldValue)
+					op += makePromCounter(promLabel, count)
+				case "gauge":
+					value := parseGauge(fieldValue)
+					op += makePromGauge(promLabel, value)
+				case "untype":
+					op += makePromUntype(promLabel, fieldValue)
+				}
+			}
+			data = data + op
```
Comment (manishasalve) [inline src/prometheus_handler.go:137]: Indentation needs to be corrected here.
Please run gofmt on this patch.

**Review by manishasalve — COMMENTED** [review]
Few minor suggestions.
Once these are completed, we can merge the branch.

**Comment by sundereshwar** [comment]
Pushed a Commit to resolve the above suggesstions

**Review by manishasalve — APPROVED** [review]

**Pull request merged**